src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,22 +15,76 @@ redirects:
   - /docs/apm/apm-ui-pages/monitoring/apm-data-infrastructure
 ---
 
-import infrastructureInfraApmDataChart from 'images/infrastructure_screenshot-crop_infra-apm-data-chart.webp'
+import apmCriticalViolationsinApdex from 'images/apm_screenshot-full_critical-violations-in-Apdex.webp'
+
+import apmCPUPercentageViolationinAPM from 'images/apm_screenshot-crop_CPU-percentage-violation-in-APM.webp'
+
+import apmToggleBetweenHistogramViews from 'images/apm_screenshot-crop_toggle-between-histogram-views.webp'
+
+import apmExploringaSpecificHostinAPM from 'images/apm_screenshot-full_exploring-a-specific-host-in-APM.gif'
+
+import apmFilterbyApplication from 'images/apm_screenshot-full_filter-by-application.webp'
 
 The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data lets you see your APM data and infrastructure data side by side so you can find the root cause of problems more quickly.
```

Comment:
This intro can be a little bit neater. I would look at M's original message and maybe string a narrative about correlating app performance with infra health. Root cause is one part of this, but I think we can get more detailed with pointing out the 'app deployment --> affects resources' vs 'infra resources --> affects app performance'

Reply:
Fixed this!